injector/dns_disruption.go

```diff
@@ -206,7 +206,12 @@ func (i *DNSDisruptionInjector) Clean() error {
 	// Remove the net_cls classid for cgroup v1
 	if !i.config.Cgroup.IsCgroupV2() {
 		if err := i.config.Cgroup.Write("net_cls", "net_cls.classid", "0"); err != nil {
-			return fmt.Errorf("unable to write net_cls classid: %w", err)
+			if os.IsNotExist(err) {
+				i.config.Log.Warnw("unable to find target container's net_cls.classid file, we will assume we cannot find the cgroup path because it is gone", "targetContainerID", i.config.TargetContainer.ID(), "err, err")
```

Comment:
```suggestion
				i.config.Log.Warnw("unable to find target container's net_cls.classid file, we will assume we cannot find the cgroup path because it is gone", "targetContainerID", i.config.TargetContainer.ID(), "err", err)
```